packages/contracts/test/sdk/sudoswap/single-token/erc721.test.ts

```diff
@@ -0,0 +1,72 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signer-with-address";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { parseEther } from "@ethersproject/units";
+
+import * as Sudoswap from "../../../../../sdk/src/sudoswap";
+
+import "@nomiclabs/hardhat-ethers";
+
+import { getChainId } from "../../../utils";
+
+describe("Sudoswap - SingleToken Erc721", () => {
```

Comment:
All above ERC1155 test comments apply here as well.

Reply:
just to clarify, the two tests aren't for ERC721 vs ERC1155. 

`sdk/sudoswap/single-token/erc721.test.ts` tests additions to the sdk. 

`test/router/v6/sudoswap/offers.test.ts` tests the sudoswap router module, as well as some components from the sdk. 

anyway, point taken - i'll make the same updates to both tests